### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by hardiksharma11**
Updated the README.md

### CODE REVIEW

**Review by vatsa287 — DISMISSED** [review]

**Comment by vatsa287** [comment]
@hardiksharma11 
https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification
Go through this for signing commits.

File: README.md [hunk 1]
```diff
@@ -11,10 +11,20 @@ To build the SDK and see changes, follow these steps:
...
+     ```bash
+     pip install setuptools
+
+3. Install dependencies and setup modules using setup.py :
 
      ```bash
-     python setup.py
+     python setup.py install
+
+## Required Dependencies
```
Comment (belyrs) [inline README.md:54]: There should be clear and straightforward documentation on how to install these required dependencies.

File: README.md [hunk 2]
```diff
@@ -11,10 +11,20 @@ To build the SDK and see changes, follow these steps:
    git clone <repository_url>
    cd <repository_directory>
 
-2. Install dependencies and setup modules using setup.py :
+2. Make sure you have python installed. You can download it from [here](https://www.python.org/downloads/). After that install setuptools using:
+     ```bash
+     pip install setuptools
```
Comment (belyrs) [inline README.md]: When I tried running it, additional pip installs were needed. Please review and update accordingly.

**Review by belyrs — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (hardiksharma11) [reply README.md:54]: I have added links of the official docs of the required dependencies .

File: README.md (hunk 2 above)
Comment (hardiksharma11) [reply README.md]: I coudn't find any other dependency other than setuptools for running setup.py. However it could differ from one system to other.

**Comment by hardiksharma11** [comment]
@adi-a11y @vatsa287 I have added prerequisites for the installation of the SDK.

**Review by belyrs — APPROVED** [review]

**Pull request merged**